Sources/Web3Core/EthereumNetwork/RequestParameter/RequestParameter+Encodable.swift

```diff
@@ -8,23 +8,23 @@
 import Foundation
 
 extension RequestParameter: Encodable {
-    /**
-     This encoder encodes `RequestParameter` associated value ignoring self value
-
-     This is required to encode mixed types array, like
-
-     ```swift
-     let someArray: [RequestParameter] = [
-        .init(rawValue: 12)!,
-        .init(rawValue: "this")!,
-        .init(rawValue: 12.2)!,
-        .init(rawValue: [12.2, 12.4])!
-     ]
-     let encoded = try JSONEncoder().encode(someArray)
-     print(String(data: encoded, encoding: .utf8)!)
-     //> [12,\"this\",12.2,[12.2,12.4]]`
-     ```
-     */
+
+    /// This encoder encodes `RequestParameter` associated value ignoring self value
+    ///
+    /// This is required to encode mixed types array, like
+    ///
+    /// ```swift
+    /// let someArray: [RequestParameter] = [
+    ///     .init(rawValue: 12)!,
+    ///     .init(rawValue: "this")!,
+    ///     .init(rawValue: 12.2)!,
+    ///     .init(rawValue: [12.2, 12.4])!
+    /// ]
+    /// let encoded = try JSONEncoder().encode(someArray)
+    /// print(String(data: encoded, encoding: .utf8)!)
+    /// //> [12,\"this\",12.2,[12.2,12.4]]`
+    /// ```
+    /// - Parameter encoder: The encoder to write data to.
```

Comment:
This format of multiline documentation (using `///` instead of `/** */`) is even used by Apple.
And to me personally, it looks a lot easier to edit.

Reply:
While i have no pref about the doc style, i have to say, that this is true only if you're staying within Xcode. All third party editor knows nothing about `///` comments, and not prefix it on new line hit.

Actually because i did wrote those comments in Sublime Text i'd choose the given one doc style formatting.